src/Monolog/Logger.php

```diff
@@ -140,6 +140,8 @@ class Logger implements LoggerInterface, ResettableInterface
     protected $exceptionHandler;
 
     /**
+     * @psalm-param array<callable(array): array>
```

Comment:
This is missing a variable name for which $param this applies to

Reply:
fixed.